server/views/partials/openConditionsBanner.html

```diff
@@ -4,7 +4,8 @@
 (data.openConditions.sexualOffences and data.openConditions.sexualOffences.haveTheyBeenEverConvicted === 'Yes' and data.openConditions.sexualOffences.canTheRiskBeManaged === 'No') or
 (data.openConditions.riskOfHarm and data.openConditions.riskOfHarm.harmManaged === 'No') or
 (data.openConditions.furtherCharges and data.openConditions.furtherCharges.increasedRisk === 'Yes') or
-(data.openConditions.riskLevels and data.openConditions.riskLevels.likelyToAbscond === 'Yes')
+(data.openConditions.riskLevels and data.openConditions.riskLevels.likelyToAbscond === 'Yes') or
+(data.openConditions.victimContactScheme.contactedVLO === 'No')
```

Comment:
(data.openConditions.victimContactScheme and data.openConditions.victimContactScheme.contactedVLO === 'No')